decisions/20230125-pcui-versioning-strategy.md

```diff
@@ -0,0 +1,16 @@
+# PCUI Versioning strategy
+
+- Status: accepted
+- Deciders: Nuraghe Team
+- Tags: versioning, pcui
+
+## Context
+We want to avoid confusing for customers using PC UI and Parallel Cluster.
```

Comment:
nit-pick: typo "to avoid confusing for customers" I think you forgot something

Reply:
Done in next rev